uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Service/JobContract.php

```diff
@@ -0,0 +1,107 @@
+<?php
+
+class CRM_HRLeaveAndAbsences_Service_JobContract {
```

Comment:
Same here, documentation can really help us new comers get around code and understand what it's for.